kafka_topic_dumper/main.py

```diff
@@ -114,7 +130,12 @@ def main():
                 if not dump_id:
                     dump_id = kafka_client.find_latest_dump_id(bucket_name)
                     logger.info('Using latest dump id <{}>'.format(dump_id))
+                cl = get_transformer_class(transformer)
```

Comment:
We should use more descriptive variable names. Something like `transformer_class`